### PR TITLE
Cypress test. Show attached files when creating task.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Add Cypress test for issue #2661 

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Displaying attached files when creating a task.', () => {
+    const issueId = '2661';
+    const labelName = `Issue ${issueId}`;
+    const imagesCount = 6;
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js:10]: imagesCount = 5

File: tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    const height = 800;
+    const posX = 10;
+    const posY = 10;
+    const color = 'gray';
+    const archiveName = `${imageFileName}.zip`;
+    const archivePath = `cypress/fixtures/${archiveName}`;
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+    let imageListToAttach = [];
+    for (let i = 1; i < imagesCount; i++) {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js:22]: i = 0

File: tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    describe(`Testing "${labelName}"`, () => {
+        it('Attach a files. Attached files is visible.', () => {
+            cy.get('input[type="file"]').attachFile(archiveName, { subjectType: 'drag-n-drop' });
+            cy.get('.cvat-file-manager-local-tab').should('contain', archiveName).and('be.visible');
+        });
+
+        it('Attach more then 4 files. Attached files is visible and contain text "<count_files> files selected".', () => {
+            cy.get('input[type="file"]').attachFile(imageListToAttach, { subjectType: 'drag-n-drop' });
+            cy.get('.cvat-file-manager-local-tab')
+                .should('contain', `${imagesCount - 1} files selected`)
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js:43]: imagesCount

File: tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js]: Done.

**Review by bsekachev — APPROVED** [review]

File: tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js]: cy.imageGenerator() generates images by adding an id starting with one. For example:
```image_issue_2661_1.png, image_issue_2661_2.png, etc.```. 
If change this value, the image with the number zero will not be found. Rewritten the loop in the form 
```for (let i = 1; i <= imagesCount; i++) {```

File: tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js (hunk 3 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js]: Done.

**Pull request merged**